tests/sessions_tests/tests.py

```diff
@@ -775,6 +775,36 @@ def test_session_delete_on_end_with_custom_domain(self):
             str(response.cookies[settings.SESSION_COOKIE_NAME])
         )
 
+    @override_settings(SESSION_COOKIE_PATH='/example/')
+    def test_session_delete_on_end_with_custom_path(self):
```

Comment:
Did you consider combining this with the above test? Seems like it would be okay but maybe there's something I'm missing.